corehq/apps/case_migrations/migration.py

```diff
@@ -1,12 +1,13 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.form_processor.backends.sql.dbaccessors import CaseReindexAccessor, iter_all_ids_chunked
 
 
 def perform_migration(domain, case_type, migration_xml):
-    case_ids = CaseAccessors(domain).get_case_ids_in_domain(case_type)
```

Comment:
What do you think of removing `get_case_ids_in_domain` or moving it to a test utility module? Seems like a bad idea to write any non-test code that uses it since it does not scale for large domains.

Reply:
yes that would be ideal and was my original intention but I wasn't able to completely remove it this time